scripts/lib/setup_venv.py

```diff
@@ -95,20 +94,32 @@ def get_venv_dependencies(vendor: str, os_version: str) -> List[str]:
     else:
         raise AssertionError("Invalid vendor")
 
+
 def install_venv_deps(pip: str, requirements_file: str) -> None:
     pip_requirements = os.path.join(ZULIP_PATH, "requirements", "pip.txt")
-    run([pip, "install", "--force-reinstall", "--require-hashes", "--requirement", pip_requirements])
+    run(
+        [
+            pip,
+            "install",
+            "--force-reinstall",
+            "--require-hashes",
+            "--requirement",
+            pip_requirements,
+        ],
+    )
```

Comment:
This transformation one a 1-line shell command into 10 lines is pretty terrible.  Any ideas for how to not do that that doesn't make the code worse?  I suppose `pip_requirements` is probably an overly long variable name that might be what pushes us over the edge here.

Reply:
`--requirement` → `-r` is one.